pom.xml

```diff
@@ -79,7 +79,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <declared.ozone.version>${ozone.version}</declared.ozone.version>
 
     <!-- Apache Ratis version -->
-    <ratis.version>0.6.0-cac3336-SNAPSHOT</ratis.version>
+    <ratis.version>0.6.0-6ab75ae-SNAPSHOT</ratis.version>
```

Comment:
Is this a duplicate of HDDS-3350? Let's decouple the Ratis version update from the rest of the changes.

Reply:
Ratis snapshot version update is required for compilation. Once HDDS-3654 is committed, I'll remove these changes.

Reply:
Marking it as pending until 3654 is committed.